kfs-ld/src/main/resources/edu/arizona/kfs/workflow/YearEndSalaryExpenseTransferDocument.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>YEST</name>
+      <parent>LDYE</parent>
+      <label>Year End Salary Expense Transfer</label>
+      <docHandler>${kfs.url}/laborYearEndSalaryExpenseTransfer.do?methodToCall=docHandler</docHandler>
+      <helpDefinitionURL>default.htm?turl=WordDocuments%2Fyearendedocs.htm</helpDefinitionURL>
+      <active>true</active>
+      <routingVersion>2</routingVersion>
+      <routePaths>
+        <routePath>
+          <start name="AdHoc" nextNode="Account" />
+          <role name="Account" nextNode="AccountingOrganizationHierarchy" />
```

Comment:
Account route node is being added in UAF-505. Remove for now. See delivered workflow for document LDYE.